user/models.py

```diff
@@ -1,11 +1,14 @@
 import uuid
 from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django_resized import ResizedImageField
 
 
 class User(AbstractUser):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     enable_notifications = models.BooleanField(default=True)
+    avatar = ResizedImageField(
+        size=[300, 300], upload_to='user', blank=False, null=True)
```

Comment:
@usmandap could you please explain if *'user'* is a s3 bucket's folder here ? If so then i think it needs to be in configuration file

Reply:
@abdulmoizeng why does this need to be in config?

Reply:
@mashhoodr keeping folder structures in config will help us re using those in some other places (if required in future requests)